### PR TITLE
Added Explode validator documentation

### PR DESCRIPTION
**Pull request by renanliberato**
I've wrote the basic documentation for Explode validator.

It came useful for me lately so I'd like to contribute with it.

It fixes both #171 and #137 , which are duplicates.

### CODE REVIEW

File: doc/book/validators/explode.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Explode Validator
+
+`Zend\Validator\Explode` executes a validator for each item exploded.
+
+## Supported options
+
+The following options are supported for `Zend\Validator\Explode`:
+
+- `valueDelimiter`: Defines the delimiter used to explode the value to an array. It defaults to `,`. If working with an array, this option isn't used.
```
Comment (froschdesign) [inline doc/book/validators/explode.md]: @renanliberato 
> If working with an array, this option isn't used.

We should clarify that the given "value" is meant here.

File: doc/book/validators/explode.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+$inArrayValidator = new Zend\Validator\InArray([
+    'haystack' => [1, 2, 3, 4, 5, 6]
+]);
+
+$explodeValidator = new Zend\Validator\Explode([
+    'validator' => $inArrayValidator
+]);
+
+$value  = [1, 4, 6, 8];
+$return = $valid->isValid($value);
```
Comment (froschdesign) [inline doc/book/validators/explode.md]: `$valid` is wrong here.

I would change the example:

```php
$explodeValidator->isValid([1, 4, 6]);    // returns true
$explodeValidator->isValid([1, 4, 6, 8]); // returns false
```

File: doc/book/validators/explode.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+// returns false
```
Comment (froschdesign) [inline doc/book/validators/explode.md]: Can be deleted.

File: doc/book/validators/explode.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+```
+
+The above example returns `true` if all $value items are between 1 and 6.
```
Comment (froschdesign) [inline doc/book/validators/explode.md]: Can be deleted, because is covered now in the example.

Comment (froschdesign) [reply doc/book/validators/explode.md]: Please add another example which demonstrate the usage of `valueDelimiter`. Thanks!

**Comment by renanliberato** [comment]
Hi @froschdesign ,

I added the valueDelimiter example and improved the valueDelimiter description.

I also appreciated your suggestions and incorporated them into the chapter.

**Comment by weierophinney** [comment]
Thanks, @renanlibrato!

**Pull request merged**